app/src/main/java/org/mozilla/focus/settings/SettingsFragment.java

```diff
@@ -4,173 +4,64 @@
 
 package org.mozilla.focus.settings;
 
-import android.app.Fragment;
-import android.content.Context;
 import android.content.Intent;
 import android.content.SharedPreferences;
 import android.content.res.Resources;
 import android.os.Bundle;
 import android.preference.ListPreference;
 import android.preference.Preference;
-import android.preference.PreferenceFragment;
 import android.preference.PreferenceScreen;
 import android.support.annotation.Nullable;
 import android.text.TextUtils;
-import android.view.Menu;
-import android.view.MenuInflater;
-import android.view.MenuItem;
-
 import org.mozilla.focus.R;
 import org.mozilla.focus.activity.InfoActivity;
 import org.mozilla.focus.activity.SettingsActivity;
 import org.mozilla.focus.autocomplete.AutocompleteSettingsFragment;
 import org.mozilla.focus.locale.LocaleManager;
 import org.mozilla.focus.locale.Locales;
-import org.mozilla.focus.search.MultiselectSearchEngineListPreference;
-import org.mozilla.focus.search.RadioSearchEngineListPreference;
-import org.mozilla.focus.search.SearchEngineManager;
 import org.mozilla.focus.telemetry.TelemetryWrapper;
 import org.mozilla.focus.widget.DefaultBrowserPreference;
 
 import java.util.Locale;
-import java.util.Set;
 
-public class SettingsFragment extends PreferenceFragment implements SharedPreferences.OnSharedPreferenceChangeListener {
-    public static final String SETTINGS_SCREEN_NAME = "settingsScreenName";
-    public static final int ALPHA_ENABLED = 255;
-    public static final int ALPHA_DISABLED = 130;
+public class SettingsFragment extends BaseSettingsFragment implements SharedPreferences.OnSharedPreferenceChangeListener {
```

Comment:
\[PMD\] INFO: The class 'SettingsFragment' has a Cyclomatic Complexity of 3 \(Highest = 7\)\.